app/controllers/syllabi_controller.rb

```diff
@@ -1,13 +1,46 @@
 class SyllabiController < ApplicationController
     def student
-      redirect_to syllabi_admin_url if current_user.role == 0
-      @students = Student.where(user_id: current_user.id)
-      @student_found = Student.find_by_id(@students[0])
+      # redirect_to syllabi_admin_url if current_user.role == 0
+      # redirect_to syllabi_reviewer_url if current_user.role == 2
+      # @students = Student.where(user_id: current_user.id)
+      # @student_found = Student.find_by_id(current_user.id)
+      if current_user.role == 0 
+        redirect_to syllabi_admin_url
+      elsif current_user.role == 2
+        redirect_to syllabi_reviewer_url
+      else #student
+        @students = Student.where(user_id: current_user.id)
+        @student_found = Student.find_by_id(@students[0])
+      end
     end
   
-    def admin
-      redirect_to root_path if current_user.role == 1
-  
-    end
-  end
-  
\ No newline at end of file
+    # def admin
+    #   redirect_to root_path if current_user.role == 1
+    #   redirect_to syllabi_reviewer_url if current_user.role == 2
+    #   @admins = Admin.where(user_id: current_user.id)
+    #   @admin_found = Admin.find_by_id(current_user.id)
+    # end
+
+    # def reviewer
```

Comment:
Will the reviewer redirect still work if this is commented out?

Reply:
It should. Look at line 7-10, I think that is what replaces what you were looking at.